src/Jackett.Common/Utils/Clients/WebClient.cs

```diff
@@ -135,15 +227,25 @@ public virtual async Task<WebClientStringResult> GetString(WebRequest request)
         protected virtual async Task<WebClientByteResult> Run(WebRequest webRequest) => throw new NotImplementedException();
 #pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
 
-        public abstract void Init();
+        public virtual void Init()
+        {
+            if (serverConfig.RuntimeSettings.IgnoreSslErrors == true)
+            {
+                logger.Info(string.Format("HttpWebClient2: Disabling certificate validation"));
```

Comment:
Yeah, I forgot about this because 3 of the 4 just say "HttpWebClient" and make no distinction on different classes, so if this is a bug, it predates this change.